### DISCUSSION TITLE
Welcome to Nebula Logger Discussions!

### DISCUSSION THREAD

**Discussion by jongpie** [jongpie/NebulaLogger#197] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by jamessimone** [comment]
Thank you, Jonathan! We appreciate everything you’ve done to make our logging lives easier

**Comment by jayakumargithub** [comment]
Jonathan, Thank you very much for this wonderful tool, it is very much needed.

**Reply by jongpie** [reply]
Thanks @jayakumargithub!!
